test/runnable/gdb15729.sh

```diff
@@ -2,12 +2,13 @@
 
 src=runnable${SEP}extra-files
 dir=${RESULTS_DIR}${SEP}runnable
-output_file=${dir}/gdb15729.sh.out
+
+set -e
```

Comment:
Flags should probably come right after the shebang, since `-u` could be added later, and those variable definitions would benefit from `-u`'s checks.

Reply:
Done.
FYI: My plan is to add `set -eu` and maybe even `set -o pipefail` by default for ALL bash scripts as that would be a sane default.